services/base-jdbc/src/main/java/org/eclipse/hono/service/base/jdbc/config/JdbcProperties.java

```diff
@@ -116,27 +120,28 @@ public static JDBCClient dataSource(final Vertx vertx, final JdbcProperties data
 
         final JsonObject config = new JsonObject()
                 .put("url", dataSourceProperties.getUrl())
-                .put("user", dataSourceProperties.getUsername());
-
-        // password is added later, after logging
-
-        if (dataSourceProperties.getDriverClass() != null) {
-            config.put("driver_class", dataSourceProperties.getDriverClass());
-        }
-        if (dataSourceProperties.getMaximumPoolSize() != null) {
-            config.put("max_pool_size", dataSourceProperties.getMaximumPoolSize());
-        }
+                .put("user", dataSourceProperties.getUsername())
+                .put("driver_class", dataSourceProperties.getDriverClass())
```

Comment:
`driver_class` is obsolete with Argoal DataSource, Argoal allows setting it but Vert.x implementation relies solely on auto recognition from JDBC URL.